examples/layers_weightnormalization.ipynb

```diff
@@ -3,7 +3,7 @@
   "nbformat_minor": 0,
   "metadata": {
     "colab": {
-      "name": "layers_weightnormalization.ipynb",
+      "name": "Copy of layers_weightnormalization.ipynb",
```

Comment:
Sean, does this name matter?

Reply:
Hmmm honestly don't know, but I'll revert it just for cleanliness